### PR TITLE
Add Operations Stage.

### PR DESCRIPTION
**Pull request by bendory**
Also: add Deployment Configuration.

### CODE REVIEW

File: docs/pipelines-terminology.md [hunk 1]
```diff
@@ -66,10 +66,18 @@ In the lists and table below, Software Artifacts includes: Documentation Source
 #### Deploy Stage
 * Semantics: Deploy software artificats and documentation to any environment other than the pipeline environment. Verify successful deployment.
 * Aliases: Install
-* Inputs: Software Source, Binary Source, Generated Software, Generated Binaries, Deployment KPIs
+* Inputs: Deployment Configuration, Software Source, Binary Source, Generated Software, Generated Binaries, Deployment KPIs
 * Outputs: Deployment Records, Deployment Reports, Secrets to access deployed resources
 * Other Results and Side Effects: Documentation hosted in a review, staging or production environment. Software running in a review, staging or production environment. Deployment KPIs might influence how the deployment proceeds; for example, for Canary or Blue/Green deployments.
 
+#### Operations Stage
```
Comment (justinabrahms) [inline docs/pipelines-terminology.md:73]: I would probably say that this is the "analyze" step under the deployment stage?

Comment (bendory) [reply docs/pipelines-terminology.md:73]: To me, the "Deployment" stage is an event -- the new assets are deployed.

"Operations Stage" is post-deployment -- continuous monitoring and analysis of deployed assets.

"Deployment Stage" ends with "Verify Successful Deployment." (And this is how it is written in this doc.)

"Operations Stage" _starts_ with "Deployment has been verified as successful" and continues until either deployment is deleted or a new artifact is deployed (at which point "Operations Stage" begins for that new artifact).

Comment (justinabrahms) [reply docs/pipelines-terminology.md:73]: Gotcha. "verify successful deployment", in my mind, includes a check of the metrics to make sure alarms aren't firing, etc.

It does make sense that the thing you're describing is the "steady state" of the pipeline. e.g. it's done it's work and it's just waiting for things to go wrong and/or new deploys to happen.

Comment (bendory) [reply docs/pipelines-terminology.md:73]: Right -- CD is a cycle, and the deployed asset remains live so long as it's in production, which I'm calling the "Operations Stage." Note that we could potentially additional stages after this, but I'll leave that for discussion:

- "Backup Stage" -- the previous one or more production assets should be in a hot backup, ready for emergency rollback if needed
- "Archive Stage" -- where we maintain some history of production assets, available for audit purposes, timeframe dependent on product requirements; these assets are older than the _n_ assets available in hot backup.

File: docs/pipelines-terminology.md [hunk 2]
```diff
@@ -66,10 +66,18 @@ In the lists and table below, Software Artifacts includes: Documentation Source
...
+* Semantics: monitoring of deployed assets
+* Aliases:
```
Comment (justinabrahms) [inline docs/pipelines-terminology.md:75]:
```suggestion
* Aliases: Steady state
```

**Review by justinabrahms — APPROVED** [review]

File: docs/pipelines-terminology.md (hunk 2 above)
Comment (bendory) [reply docs/pipelines-terminology.md]: thanks, incorporated this change + some other aliases

**Review by MarckK — APPROVED** [review]

**Pull request merged**